ADOPTERS.md

```diff
@@ -0,0 +1,36 @@
+# WAMR adopters
+
+_If you are using WAMR in production/pre-production at your organization, please add your company name to this list.
+The list is in alphabetical order._
+
+| Organization                           | Contact                                                      | Status                                                       | Description of Use                                           |
+| -------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Alibaba](https://www.alibaba.com)     | [@johnlanni](https://github.com/johnlanni)                   | ![production](https://img.shields.io/badge/-production-blue?style=flat) | Higress is a next-generation cloud-native gateway built on the core of open-source Istio + Envoy based on Alibaba's internal Envoy Gateway practice. |
+| [Amazon](https://www.amazon.com)       | [@loganek](https://github.com/loganek)                       | ![production](https://img.shields.io/badge/-production-blue?style=flat) | Amazon Prime Video is an American subscription video on-demand over-the-top streaming and rental service of Amazon offered as a standalone service or as part of Amazon's Prime subscription. |
```

Comment:
I'd remove `American` from the sentence; the service is available globally. Also, Prime Video does not only offer video on demand but also live content.

I think something like that would do:
> Prime Video is a global streaming service by Amazon that provides on-demand access to a vast library of movies, TV shows, and original programming, as well as live content.